backend/src/main/java/de/bund/digitalservice/ris/norms/application/exception/StatusNotFoundException.java

```diff
@@ -0,0 +1,33 @@
+package de.bund.digitalservice.ris.norms.application.exception;
+
+import de.bund.digitalservice.ris.norms.utils.exceptions.NormsAppException;
+import java.net.URI;
+import java.util.Map;
+import java.util.UUID;
+import lombok.Getter;
+
+/** Indicates that the requested status does not exist. */
+@Getter
+public class StatusNotFoundException extends RuntimeException implements NormsAppException {
+
+  private final UUID processId;
+
+  public StatusNotFoundException(UUID processId) {
+    this.processId = processId;
+  }
+
+  @Override
+  public URI getType() {
+    return URI.create("/errors/process-id-not-found");
+  }
+
+  @Override
+  public String getTitle() {
+    return "Could not find process with id " + processId;
```

Comment:
the title should be a static string and not depend on the paramters, the processId should instead be returned by getProperties. The message you have written here is more like the detail message and should therefore be passed to the super constructor.